solaris/data/coco.py

```diff
@@ -191,6 +195,10 @@ def geojson2coco(image_src, label_src, output_path=None, image_ext='.tif',
     for gj in tqdm(label_list):
         logger.debug('Reading in {}'.format(gj))
         curr_gdf = gpd.read_file(gj)
+        
+        if remove_all_multipolygons is True:
+            curr_gdf = remove_multipolygons(curr_gdf)
```

Comment:
Good idea. With the dataset I was working with, some multipolygons represented a single instance and some represented multiple field instances. Others were artifacts introduced by clipping. So I'll put a warning about this in the docs, it might not be valid to explode in all cases and it's good to know where the multipolygons are coming from.